modules/post/windows/gather/bloodhound.rb

```diff
@@ -0,0 +1,63 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+  include Msf::Post::Windows::Powershell
+  include Msf::Post::Windows::Priv
+  include Msf::Post::Common
+
+  def initialize(info = {})
+    super(update_info(
+      info,
+      'Name' => 'BloodHound',
+      'Description' => %q(This module load in memory Invoke-BloodHound.ps1 and executes it.),
+      'License' => MSF_LICENSE,
+      'Author' => [
+        'BloodHound Gang', # Invoke-BloodHound
+        'phra' # MSF Module
+      ],
+      'References' => [''],
```

Comment:
Link to `https://github.com/BloodHoundAD/BloodHound` perhaps?

Reply:
i will add it.